ERCS/erc-7660.md

```diff
@@ -0,0 +1,350 @@
+---
+eip: 7660
+title: Release tokens on predefined periods
+description: Controlled by smart contracts, tokens can be linearly released based on the predefined period.
+author: Ali Kaya (@AliKaya) <ali.kayatur747@gmail.com>
+discussions-to: https://ethereum-magicians.org/t/erc-7660-a-protocol-for-erc20-tokens-which-released-based-on-predefined-periods-controlled/19305
+status: Draft
+type: Standards Track
+category: ERC
+created: 2024-03-08
+requires: 20
+---
+
+## Abstract
+
+This proposal outlines [ERC-20](./eip-20.md) tokens which released based on predefined periods controlled by smart contracts,smart contract allows token released based on predefined periods, enabling lock token balances and gradually release tokens.
+
+## Motivation
+The motivation behind this protocol is to enrich the ERC repository. By allowing [ERC-20](./eip-20.md) tokens to have custom linear release schedules, lock-up mechanisms, and whitelisting, this protocol can find applications in scenarios such as MEME tokens, games, DAO organizations, and options trading.
+
+## Specification
+
+This protocol introduces improvements in the following areas:
+
+1. [ERC-20](./eip-20.md) Tokens and Linear Release Functionality.
+2. Online Representation of [ERC-20](./eip-20.md)  Balances.
+3. Lock-Up Functionality for [ERC-20](./eip-20.md) Tokens.
+4. "Sorter" Algorithm.
+5. Low-Cost Swap Algorithm.
```

Comment:
Your proposal doesn't actually specify anything. Normally ERCs include at least a Solidity interface, and associated documentation.